<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="aprobados">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Créditos Aprobados: XX</string>
   </property>
  </widget>
  <widget class="QLabel" name="cursados">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Créditos Cursando: XX</string>
   </property>
  </widget>
  <widget class="QLabel" name="pendientes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Créditos Pendientes: XX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Créditos obligatorios hasta semestre N:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Semestre</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonContar1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Contar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtObli">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Créditos del semestre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonContar2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Contar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Semestre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRegresar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
